src/job.ts

```diff
@@ -107,6 +124,21 @@ export class CronJob<OC extends CronOnCompleteCommand | null = null, C = null> {
 			throw new ExclusiveParametersError('timeZone', 'utcOffset');
 		}
 
+		if (params.errorHandler != null) {
```

Comment:
also a superfluous null check. moreover, this will ignore the `if` block below, which will lead to undesired behavior. you should add the `errorHandler` param on every `return new CronJob` inside the `if` block instead.